config/locales/en.yml

```diff
@@ -1110,7 +1110,7 @@ en:
   default_role_anonymous: "Anonymous"
   default_role_developer: "Developer"
   default_role_project_admin: "Project admin"
-  default_role_project_creator: "Project creator"
+  default_role_project_creator_and_staff_manager: "Staff manager and project creator"
```

Comment:
@wielinde I have altered the name a bit from what was discussed and agreed upon. Please take a look if you are ok with this change.

Reply:
Sorry, @wielinde I had to alter the name again as my proposal was longer than 30 chars.

Reply:
I see. "Staff and projects manager", just as you proposed, sounds good to me.